app/src/main/res/menu/options_menu.xml

```diff
@@ -1,4 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
     android:orientation="vertical" android:layout_width="match_parent"
-    android:layout_height="match_parent"/>
\ No newline at end of file
+    android:layout_height="match_parent">
+    <item
+        android:id="@+id/option_log_out_id"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:enabled="true"
+        android:onClick="displayTosatLogOut"
```

Comment:
I know this method isn't implemented yet, but I think you made a typo.  Toast not  Tosat